src/history/history-entry.entity.ts

```diff
@@ -28,14 +28,13 @@ export class HistoryEntry {
   @UpdateDateColumn()
   updatedAt: Date;
 
-  // The optional note parameter is necessary for the createNote method in the NotesService,
-  // as we create the note then and don't need to add it to the HistoryEntry.
-  public static create(user: User, note?: Note): HistoryEntry {
+  public static create(
+    user: User,
+    note: Note,
```

Comment:
I think `pinStatus` should be added here and have default value false.

Reply:
Not sure if we ever want to create a new HistoryEntry with pinStatus true directly, but I guess this could be usefull, so I'll add this.

Reply:
When new entries are created, aren't they always unpinned?